src/Symfony/Bundle/TwigBundle/DependencyInjection/TwigExtension.php

```diff
@@ -170,6 +170,18 @@ public function load(array $configs, ContainerBuilder $container)
             $container->removeDefinition('twig.cache_warmer');
             $container->removeDefinition('twig.template_cache_warmer');
         }
+
+        $this->addClassesToPreload([
+            'Symfony\Component\Stopwatch\Section',
```

Comment:
shouldn't it be added by FrameworkBundle when defining the stopwatch service instead ? Thus, I don't think we use it in prod code.